src/content/quickstart/_index.en.md

```diff
@@ -1,17 +1,68 @@
 +++
-title = "Quickstart"
+title = "Quickstart guide"
 date = 2020-02-19T20:03:44+01:00
 weight = 5
 pre = "<b>1. </b>"
 +++
 
+## Basic overview
 
-# Quickstart guide
+Submariner has two main core pieces (the broker and submariner), more information about 
+this topic can be found in the [Architecture](../architecture) section.
 
-## Installing
+#### The broker
+It's an API, to which all participating clusters are given access, and where two objects are exchanged via CRDs:
+* Cluster(.submariner.io): defines a participating cluster and it's IP CIDRs.
+* Endpoint(.submariner.io): defines a connection endpoint to a Cluster, and the reachable cluster IPs from such endpoint.
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
```

Comment:
Is this Latin? Where did that come from ...

Reply:
It's a very old latin text used commonly in web design to fill up spaces and see how it looks: https://en.wikipedia.org/wiki/Lorem_ipsum :-)